zen/.idea/inspectionProfiles/profiles_settings.xml

```diff
@@ -0,0 +1,7 @@
+<component name="InspectionProjectProfileManager">
+  <settings>
+    <option name="useProjectProfile" value="false" />
+    <option name="USE_PROJECT_PROFILE" value="false" />
+    <version value="1.0" />
+  </settings>
+</component>
```

Comment:
this is an IDE file, I would recommend you add `.idea` to your own global gitignore file